apps/vs-code-designer/src/package.json

```diff
@@ -67,6 +67,15 @@
           "dark": "assets/dark/CreateNewProject.svg"
         }
       },
+      {
+        "command": "azureLogicAppsStandard.createNewCodeProject",
+        "title": "Create New Custom Logic App Workspace...",
+        "category": "Azure Logic Apps",
+        "icon": {
+          "light": "assets/light/CreateNewCodeProject.svg",
+          "dark": "assets/dark/CreateNewCodeProject.svg"
+        }
+      },
       {
         "command": "azureLogicAppsStandard.createCodeless",
         "title": "Create Workflow...",
```

Comment:
```suggestion
        "title": "Create workflow...",
```

Reply:
> 

Was this comment intentional? This command is not part of my project